src/interface/src/app/top-bar/top-bar.component.html

```diff
@@ -63,16 +63,15 @@
       target="_blank"
       aria-label="help button"
       rel="noopener noreferrer"
```

Comment:
nit: can we make the spacing between the top bar components a little more consistent? ("Feedback" is kind of far away from the ? icon)

Reply:
Good call, thank you!
<img width="481" alt="Screen Shot 2023-09-18 at 15 48 27" src="https://github.com/OurPlanscape/Planscape/assets/358892/b57ff726-1575-4699-b02c-af9160475035">